snowshu/core/main.py

```diff
@@ -116,28 +117,22 @@ def source(filename):
     default=DEFAULT_RETRY_COUNT
 )
 @click.option(
-    '--architecture', '-arch',
-    help="Select arch for replicas, use as 'snowshu create -arch arm64 -arch amd64'",
-    default=None,
-    multiple=True
+    '--multiarch', '-m',
+    help="Tells SnowShu to build replicas of all architectures",
```

Comment:
We should probably keep the list of architectures in the help bit for reference since it isn't a long list currently.